.github/workflows/build-java.yml

```diff
@@ -0,0 +1,67 @@
+name: "Generate Java SDK"
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Run generation nightly on latest OpenAPI spec
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+          cache: maven
+
+      - uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '8.x.x'
```

Comment:
```suggestion
      - uses: microsoft/setup-kiota@v0.5.0
```

Reply:
I'm happy to go in the direction you are proposing, but, currently, the installation of the tools is based on [`dotnet tool install`](https://github.com/octokit/source-generator/blob/main/scripts/install-tools.sh), if we move to the specific GH Action it would be best to update all of the jobs.
Looking for the maintainers of this project guidance here.